libcxx/test/std/numerics/simd/simd.class/subscript.assert.pass.cpp

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <simd>
+
+// Test hardening assertions for std::datapar::simd.
```

Comment:
This assertion is actually not hardened in the Standard. We should move it to `libcxx/test/libcxx`. If the spec said `Hardened Precondition`, then it would be a hardened one.